### ISSUE TITLE
Simply change the title of all links across ulogs.org to show ulogs

### ISSUE THREAD

**Issue by surpassinggoogle** [ulogs/ulogs#9]
Some pages or many pages on ulogs.org still show busy instead of ulogs. you will notice this if you did a search in google using the search query link:ulogs.org

In this task, kindly tend to all such cases across ulogs.org so that all pages across the website are titled ulogs instead of busy

Note: ulogs.org is a fork of busy

**Comment by sweetcha** [comment]
Yes, I notice it shows busy but the good thing is I always use busy to create my post before and it's easy for me to use ulogs.org. Your doing such a great job, good luck and more power.😊
